experiment/autobumper/bumper/bumper.go

```diff
@@ -376,8 +378,11 @@ func (w HideSecretsWriter) Write(content []byte) (int, error) {
 // "images" contains the tag replacements that have been made which is returned from "updateReferences([]string{"."}, extraFiles)"
 // "images" and "extraLineInPRBody" are used to generate commit summary and body of the PR
 func UpdatePR(gc github.Client, org, repo string, images map[string]string, extraLineInPRBody, login, baseBranch, headBranch string, allowMods bool, prefixes []Prefix, versions map[string][]string) error {
```

Comment:
Is this still used?

Reply:
Not in the autobumper, and it is redundant; however it is exported, and I wanted to keep the change backwards compatible.
/assign @alvaroaleman 
Can we get rid of this?

Reply:
Good thoughts!

Reply:
Yeah, we are still using this